### PR TITLE
Update wording

### PR DESCRIPTION
**Pull request by shichun-0415**
### What is changed?
Update wording

### Any related PRs or issues?
No

### CODE REVIEW

**Review by marsishandsome — APPROVED** [review]
LGTM

File: content/docs/5.1/deploy/install/production.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ For example, execute the following command to check the status of the `tikv-test
 tiup cluster display tikv-test
 ```
 
-Expected output includes the instance ID, role, host, listening port, and status (because the cluster is not started yet, so the status is `Down`/`inactive`), and directory information.
+The command output should include the instance ID, role, host, listening port, and status (the status of the cluster is `Down`/`inactive` because it is not started yet), and directory information.
```
Comment (TomShawn) [inline content/docs/5.1/deploy/install/production.md:151]: @shichun-0415 Maybe we can remove "should".

File: content/docs/5.1/deploy/install/test.md [hunk 2]
```diff
@@ -10,27 +10,27 @@ This guide describes how to install and deploy TiKV for test using TiUP playgrou
 
 ## TiUP Playground
 
-This chapter describes how to deploy a TiKV cluster using TiUP Playground.
+This chapter describes how to deploy a TiKV cluster using TiUP playground.
```
Comment (TomShawn) [inline content/docs/5.1/deploy/install/test.md:13]: TiUP Playground is a component name.

File: content/docs/5.1/deploy/operate/upgrade.md [hunk 3]
```diff
@@ -17,15 +17,14 @@ This section introduces the preparation works needed before upgrading your TiKV
 
 Before upgrading your TiKV cluster, you first need to upgrade TiUP.
 
-1. Upgrade the TiUP version. It is recommended that the TiUP version is `1.4.0` or later.
+1. Upgrade the TiUP version. It is recommended that the TiUP version be `1.4.0` or later.
```
Comment (TomShawn) [inline content/docs/5.1/deploy/operate/upgrade.md:20]:
```suggestion
It is recommended to upgrade the TiUP version to `1.4.0` or later.
```

File: content/docs/5.1/deploy/operate/upgrade.md [hunk 4]
```diff
@@ -17,15 +17,14 @@ This section introduces the preparation works needed before upgrading your TiKV
...
 
     ```bash
     tiup update --self
     tiup --version
     ```
 
-2. Upgrade the TiUP Cluster version. It is recommended that the TiUP Cluster version is `1.4.0` or later.
-
+2. Upgrade the TiUP Cluster version. It is recommended that the TiUP Cluster version be `1.4.0` or later.
```
Comment (TomShawn) [inline content/docs/5.1/deploy/operate/upgrade.md:27]: This line should be consistent with Line 20 in style?

File: content/docs/5.1/deploy/operate/upgrade.md [hunk 5]
```diff
@@ -38,18 +37,18 @@ Before upgrading your TiKV cluster, you first need to upgrade TiUP.
 Skip this step if one of the following situations applies:
 
 + You have not modified the configuration parameters of the original cluster. Or you have modified the configuration parameters using `tiup cluster` but no more modification is needed.
-+ After the upgrade, you want to use v5.0's default parameter values for the unmodified configuration items.
++ After the upgrade, you want to use default parameter values of v5.0 for the unmodified configuration items.
 {{< /info >}}
 
-1. Edit the topology file:
+1. Enter the `vi` editing mode to edit the topology file:
```
Comment (TomShawn) [inline content/docs/5.1/deploy/operate/upgrade.md:43]: I am not sure whether the following command opens the vi editor. The vi editor is usually opened by `vi` or `vim` command.

File: content/docs/5.1/deploy/operate/upgrade.md [hunk 6]
```diff
@@ -38,18 +37,18 @@ Before upgrading your TiKV cluster, you first need to upgrade TiUP.
...
 
     ```bash
     tiup cluster edit-config <cluster-name>
     ```
 
 2. Refer to the format of [topology](https://github.com/pingcap/tiup/blob/release-1.4/embed/templates/examples/topology.example.yaml) configuration template and fill the parameters you want to modify in the `server_configs` section of the topology file.
 
-3. Save the changes and exit the editing mode.
+3. After the modification, <kbd>:</kbd> + <kbd>w</kbd> + <kbd>q</kbd> to save the change and exit the editing mode. Enter <kbd>Y</kbd> to confirm the change.
```
Comment (TomShawn) [inline content/docs/5.1/deploy/operate/upgrade.md:51]: I guess the readers (developers, system administrators, etc.) have already mastered the basics of `vi` editor.
```suggestion
After the modification, press <kbd>:</kbd> + <kbd>w</kbd> + <kbd>q</kbd> to save the change and exit the editing mode. Enter <kbd>Y</kbd> to confirm the change.
```

File: content/docs/5.1/deploy/install/production.md (hunk 1 above)
Comment (shichun-0415) [reply content/docs/5.1/deploy/install/production.md:151]: Agree

File: content/docs/5.1/deploy/install/test.md [hunk 7]
```diff
@@ -206,7 +206,7 @@ To deploy a TiKV cluster with multiple nodes for test, take the following steps:
                     --log-file=tikv3.log
     ```
 
-You can use the [pd-ctl](https://github.com/pingcap/pd/tree/master/tools/pd-ctl) tool to verify whether PD and TiKV are successfully deployed:
+Use the [pd-ctl](https://github.com/pingcap/pd/tree/master/tools/pd-ctl) tool to verify whether PD and TiKV are successfully deployed:
```
Comment (TomShawn) [inline content/docs/5.1/deploy/install/test.md:209]: To me, there are some slight differences between "you can do" and the imperative "do". In this context, "verify whether PD and TiKV are successfully deployed" looks like a piece of supplementary information in addition to the steps above. It is optional. IMO, "you can use" may look better.

File: content/docs/5.1/deploy/install/test.md (hunk 2 above)
Comment (shichun-0415) [reply content/docs/5.1/deploy/install/test.md:13]: But in other places of this file, it is written as "playground". Is it necessary to unify the style?

File: content/docs/5.1/deploy/operate/upgrade.md (hunk 3 above)
Comment (shichun-0415) [reply content/docs/5.1/deploy/operate/upgrade.md:20]: Agree

File: content/docs/5.1/deploy/operate/upgrade.md (hunk 4 above)
Comment (shichun-0415) [reply content/docs/5.1/deploy/operate/upgrade.md:27]: Agree

File: content/docs/5.1/deploy/operate/upgrade.md (hunk 5 above)
Comment (shichun-0415) [reply content/docs/5.1/deploy/operate/upgrade.md:43]: Your point makes sense. But I think the command provided here is to edit the file, not to enter the vi editor.

File: content/docs/5.1/deploy/install/test.md (hunk 7 above)
Comment (shichun-0415) [reply content/docs/5.1/deploy/install/test.md:209]: Good point. Agree.

File: content/docs/5.1/deploy/operate/upgrade.md (hunk 5 above)
Comment (shichun-0415) [reply content/docs/5.1/deploy/operate/upgrade.md:43]: Maybe we can change the sentence to "Edit the topology file in 'vi' editing mode"?

File: content/docs/5.1/deploy/operate/upgrade.md (hunk 6 above)
Comment (shichun-0415) [reply content/docs/5.1/deploy/operate/upgrade.md:51]: You are right. I'll check with Mr. Gu Liangliang to decide whether to simplify the description.

File: content/docs/5.1/deploy/install/test.md (hunk 2 above)
Comment (TomShawn) [reply content/docs/5.1/deploy/install/test.md:13]: If "playground" is used in similar places, we can keep it.

**Pull request merged**